sources/Cargo.toml

```diff
@@ -150,7 +151,7 @@ members = [
 
     "shimpei",
 
-    "xfscli",
+    "xfscli"
```

Comment:
not sure if your IDE removed this -- toml accepts commas on the last item of a list and I tend to prefer that as it allows you to add items after the last item without needing to add a comma

Reply:
I find that in our projects I basically need to turn off all of my IDE's automatic reformatting with the exception of Rust files where it should be using Rust Fmt on save. If your IDE is reformatting TOML and Markdown you will get these unnecessary perturbations in your PRs. :+1:

Reply:
nit: you still have a change here where one is not needed.